starters/cms/src/data.ts

```diff
@@ -174,7 +212,39 @@ export async function syncExistingCollection(
             return { didSync: false }
         }
 
-        await syncCollection(collection, dataSource, existingFields, slugField)
+        const fields: ManagedCollectionFieldInput[] = []
+        for (const field of existingFields) {
+            if (field.type === "multiCollectionReference" || field.type === "collectionReference") {
+                fields.push({
+                    id: field.id,
+                    name: field.name,
+                    type: field.type,
+                    collectionId: field.collectionId,
+                })
+            } else if (field.type === "enum") {
+                fields.push({
+                    id: field.id,
+                    name: field.name,
+                    type: field.type,
+                    cases: [],
+                })
+            } else if (field.type === "file" || field.type === "image") {
+                fields.push({
+                    id: field.id,
+                    name: field.name,
+                    type: field.type,
+                    allowedFileTypes: [],
```

Comment:
The empty arrays are incorrect

Reply:
Yo, sorry it's a bit hard to solve this since i can't find documentation, but here is [the commit](https://github.com/framer/plugins/pull/222/commits/f7482e4f80c6195822e56605d50b3e4e923c95d6)